doc/functions/parent.rst

```diff
@@ -17,4 +17,22 @@ parent block when overriding a block by using the ``parent`` function:
 The ``parent()`` call will return the content of the ``sidebar`` block as
 defined in the ``base.html`` template.
 
-.. seealso:: :doc:`extends<../tags/extends>`, :doc:`block<../functions/block>`, :doc:`block<../tags/block>`
+``parent()`` can also be used in conjunction with the :doc:`block<../tags/use>` 
+tag, in which case it seems to pull from the most recently defined parent block: 
```

Comment:
Agreed.  I only wrote "seems to" because there is some uncertainty on my part, and I couldn't find documentation to resolve my uncertainty.  If you know more than me and know that there is no uncertainty in that behavior, then please, remove "seems to".   I wrote "seems to" because that is what I observed, without having more strongly sourced knowledge.